### PR TITLE
Style Convention `.md`

### PR DESCRIPTION
**Pull request by robbykap**

### CODE REVIEW

File: quality_code.md [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+
+## Functions
+
+- Break your code into functions.
+- Each function should do one thing.
+- Each function should be small and focused.
+
+```c++
+// Good
+double calculate_area_of_circle(int radius) {
```
Comment (jmc8290) [inline quality_code.md:274]: I think we should delete this section. This feels overly excessive.

File: quality_code.md [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+
+In this class, you are encouraged to use whichever case you prefer but make sure to be consistant from program to program. 
+
+### Snake case
+
+Is when you use lowercase letters and separate words with underscores. For example:
+
+```c++
+// Snake case 
+auto this_is_an_variable = 'This is an variable';
```
Comment (jmc8290) [inline quality_code.md:21]: switch to this is a variable

File: quality_code.md [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+}
+```
+
+### Camel case
+
+Is when you use lowercase letters for the first word and uppercase letters for the first letter of each subsequent word. For example:
+
+```c++
+// Camel case
+auto thisIsAnVariable = 'This is an variable';
```
Comment (jmc8290) [inline quality_code.md:34]: switch to "this is a variable"

File: quality_code.md [hunk 4]
```diff
@@ -0,0 +1,346 @@
...
+
+## Case
+
+There are many different ways to name things in programming. For example, you can use snake case or camel case. 
+
+In this class, you are encouraged to use whichever case you prefer but make sure to be consistant from program to program. 
+
+### Snake case
+
+Is when you use lowercase letters and separate words with underscores. For example:
```
Comment (byubean) [inline quality_code.md:17]: > Snake case is when you...

File: quality_code.md [hunk 5]
```diff
@@ -0,0 +1,346 @@
...
+std::string pie_flavor = 'key lime';
+
+void get_flavors() {
+    // code here
+}
+```
+
+### Camel case
+
+Is when you use lowercase letters for the first word and uppercase letters for the first letter of each subsequent word. For example:
```
Comment (byubean) [inline quality_code.md:30]: > Camel case is when...

File: quality_code.md [hunk 6]
```diff
@@ -0,0 +1,346 @@
...
+
+# Style
+
+In this section, we will cover the style conventions for writing C++ code. These conventions are important because they help you write code that is easy to read and understand. When you write code that is easy to read and understand, it is easier to debug and maintain. Aspects of style that we will cover include: naming conventions, brackets, and whitespace.
+
+## Case
+
+There are many different ways to name things in programming. For example, you can use snake case or camel case. 
+
+In this class, you are encouraged to use whichever case you prefer but make sure to be consistant from program to program.
```
Comment (byubean) [inline quality_code.md:13]: > Is is common to see code in C++ using either snake case or camel case. You may use whichever convention you prefer, but make sure to be consistent.

File: quality_code.md [hunk 7]
```diff
@@ -0,0 +1,346 @@
...
+# Good
+def calculate_area_of_circle(int radius) {
+    return 3.14 * radius ** 2
+}
+
+# Bad
+def calculate_area_of_circle(int foo) {
+    return 3.14 * foo ** 2
+}
+```
```
Comment (byubean) [inline quality_code.md:135]: Double `**` is not C++. :)

File: quality_code.md [hunk 8]
```diff
@@ -0,0 +1,346 @@
...
+// REALLY BAD
+void function1() {
+    // code here
+}
+
+
+void function2() {
+    // code here
+}
+
```
Comment (byubean) [inline quality_code.md:198]:
```c++
int function1() { return 7;} char function2() {
return 'c';  
   }


void function3() 
{ /* code */

}

```

File: quality_code.md [hunk 9]
```diff
@@ -0,0 +1,346 @@
...
+#include <iostream>
+using std::cout;
+using std::endl;
+
+#include "where.h"
+
+void swap(int &first, int &second)
+// void swap(int first, int second)
+{
```
Comment (byubean) [inline quality_code.md:306]: remove the commented code.

**Review by byubean — APPROVED** [review]

**Pull request merged**